test/images/vulnerability-scan/scan.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The Pipeline Service Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+
+images=(
+ "access-setup"
+ "argocd-registrar"
+ "cluster-setup"
+ "gateway-deployment"
+ "kcp-registrar"
+ "shellcheck"
+)
+
+get_digest() {
+  #number of retries=3
+  for _ in {1..3}
+  do
+    #Get the digest of the image
+    digest_http_code=$(curl -s -w '%{http_code}' -o /tmp/digest.json -H "Accept: application/vnd.quay.distribution.manifest.v2+json Content-type: application/json Authorization: Bearer           $AUTH_BEARER_TOKEN " https://quay.io/api/v1/repository/redhat-pipeline-service/"$img")
```

Comment:
```suggestion
    digest_http_code=$(curl -s -w '%{http_code}' -o /tmp/digest.json -H "Accept: application/vnd.quay.distribution.manifest.v2+json Content-type: application/json Authorization: Bearer $AUTH_BEARER_TOKEN " https://quay.io/api/v1/repository/redhat-pipeline-service/"$img")
```
I'm not sure if those spaces were intended.